tmui/src/app/src/core/tool/LabelLayer.ts

```diff
@@ -77,10 +77,13 @@ abstract class LabelLayer extends VectorTileLayer {
             var geomType = feature.getGeometry().getType();
             var label = feature.get('label');
             var fillColor: ol.Color;
-            if (label !== undefined) {
-                fillColor = this.colorMapper(label).toOlColor();
+            if (label == "None") {
```

Comment:
```suggestion
            if (label === "None") {
```

I'm no js/typescript expert, but probably here a triple-equal check is useful: https://stackoverflow.com/questions/359494/